pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, buildPackages, removeReferencesTo, addDriverRunpath, pkg-config, perl, texinfo, texinfo6, yasm
+{ lib, config, stdenv, buildPackages, removeReferencesTo, addDriverRunpath, pkg-config, perl, texinfo, texinfo6, yasm
```

Comment:
Many packages have a `cudaSupport ? config.cudaSupport`, i.e. `cudaSupport` is also a top-level parameter. Not sure which pattern is preferred.

Reply:
I think adding that as an option would be a bit confusing, as we already support some cuda features on our full FFmpeg version.

Reply:
True, usually "enable CUDA support" is a binary switch, but ffmpeg can use CUDA with only free components, or with unfree ones. :confused: